workflows/edit_data_packages/obsolescence_chain.Rmd

```diff
@@ -0,0 +1,22 @@
+
+## Obsolescence chain
```

Comment:
Can rename to `Set obsolescence`?

I think this is a pretty rare use case. Obsolescence should be handled with `publish_update()` >90% of the time. I think it is dangerous to do this by hand unless absolutely necessary. Maybe explain possible reasons why this would be necessary and make sure to explain that `publish_update` and `update_object` will do automatically with cross-references.

Reply:
Not sure how to start a comment on another section since your comment disappeared after I edited the file but I included the publish_update cross-reference that you commented in the "Add a pre generated identifier to the eml"  but I can't tell if it is working. Are the links for them just [publish update](#publish-update) and [update object](#update-object) ?